Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM node:18.16.0-alpine3.17@sha256:44aaf1ccc80eaed6572a0f2ef7d6b5a2982d54481e4255480041ac92221e2f11
+# syntax=docker/dockerfile:1.4
 
-RUN apk --no-cache add zip
+FROM node:18-alpine@sha256:9036ddb8252ba7089c2c83eb2b0dcaf74ff1069e8ddf86fe2bd6dc5fecc9492d
+
+RUN apk add --no-cache zip
```

Comment:
Hmm, `zip` should exist in the image already.  Did you run into a problem with it being missing?  Possibly related to changing the base image.